OAIMetadataFormat_JATS.inc.php

```diff
@@ -391,7 +391,7 @@ protected function _mungeMetadata($doc, $journal, $article, $section, $issue) {
 		foreach (['volume', 'issue'] as $nodeName) {
 			$match = $xpath->query("//article/front/article-meta/$nodeName");
 			if ($match->length) {
-				$match->item(0)->setAttribute('seq', array_search($article->getId(), $articleIds)+1);
+				$match->item(0)->setAttribute('seq', ((int) $publication->getData('seq')) + 1);
```

Comment:
It looks like `articleIds` could be removed above, since we're no longer using it here.

Reply:
Updated and removed in all three branches.